src/App.js

```diff
@@ -5,10 +5,14 @@ import Mockman from "mockman-js";
 import { Footer, Header } from "./components";
 import { Auth } from "./pages/auth/Auth";
 import { NotAuth } from "./pages/auth/NotAuth";
```

Comment:
You can have one index.js in pages and there export all the pages, then here you can import all pages in one line.

Reply:
sure, will add